src/extensions/extra/HTMLHinter/nls/root/strings.js

```diff
@@ -16,7 +16,7 @@ define({
     "INVALID_CSS_RULE": "<p><em data-highlight='[[cssRule.start]],[[cssRule.end]]'>This</em> CSS rule is not legal CSS.</p>",
     "INVALID_TAG_NAME": "<p>The <code>&lt;</code> character <em data-highlight='[[openTag.start]],[[openTag.end]]'>here</em> appears to be the beginning of a tag, but is not followed by a valid tag name.</p> <p>If you just want a <code>&lt;</code> to appear on your Web page, try using <code>&amp;lt;</code> instead.</p> <p>Or, see a <a href='https://developer.mozilla.org/en/docs/Web/Guide/HTML/HTML5/HTML5_element_list'>list of HTML5 tags</a>.</p>",
     "JAVASCRIPT_URL_NOT_ALLOWED": "<p>Sorry, but security restrictions on this site prevent you from using the <code>javascript:</code> URL <em data-highlight='[[value.start]],[[value.end]]'>here</em>. If you really need to use JavaScript, consider using <a href='http://jsbin.com/'>jsbin</a> or <a href='http://jsfiddle.net/'>jsfiddle</a>.</p>",
-    "MISMATCHED_CLOSE_TAG": "<p>The closing <code>&lt;/[[closeTag.name]]&gt;</code> tag <em data-highlight='[[closeTag.start]],[[closeTag.end]]'>here</em> doesn't pair with the opening <code>&lt;[[openTag.name]]&gt;</code> tag <em data-highlight='[[openTag.start]],[[openTag.end]]'>here</em>. This is likely due to a missing <code>&lt;/[[openTag.name]]&gt;</code> tag.",
+    "MISMATCHED_CLOSE_TAG": "<p>The closing <code data-highlight='[[closeTag.start]],[[closeTag.end]]'>&lt;/[[closeTag.name]]&gt;</code> tag doesn't pair with the opening <code data-highlight='[[openTag.start]],[[openTag.end]]'>&lt;[[openTag.name]]&gt;</code> tag. This is likely due to a missing <code>&lt;/[[openTag.name]]&gt;</code> tag.",
```

Comment:
We should figure out the right way to do these strings.  It isn't to have them in here.  cc @gideonthomas

Reply:
All of this needs to be in the thimble repo in the `/locales/en-US/editor.properties` file. The tricky thing here is that there's a lot of code in the strings and I'm not sure how to handle that (it might be ok). Let's open up a PR for these strings in Thimble and get Theo to weigh in

Reply:
out of curiosity, I thought all these strings would be in slowparse? Are we managing them in brackets instead?

Reply:
slowparse has its own template messages, but those are pretty much "tailored for nothing", so we decided to only tap into Slowparse for the actual error + parse information, and have the "messages" that present that information in brackets, since that's the product in which the actual text makes sense.